modules/nf-core/gatk4/genotypegvcfs/meta.yml

```
@@ -20,7 +20,7 @@ input:
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
-  - gvcf:
+  - input:
       type: file
       description: |
         gVCF(.gz) file or to a GenomicsDB
```

```suggestion
        gVCF(.gz) file or to a GenomicsDB
```
Or a what to a GenomicsDB?

---

yes a genomicsDB is a file format used to merge vcfs from different samples

---

ah nvm I see what you mean